README.md

```diff
@@ -5,27 +5,58 @@
 
 A comprehensive SDK for building Microsoft Teams applications, bots, and AI agents using Python. This SDK provides a high-level framework with built-in Microsoft Graph integration, OAuth handling, and extensible plugin architecture.
 
+<a href="https://microsoft.github.io/teams-ai" target="_blank">
+    <img src="https://img.shields.io/badge/📖 Getting Started-blue?style=for-the-badge" />
+</a>
+
+- [Getting Started](#getting-started)
+- [Packages](#packages)
+- [Test Apps](#test-apps)
+- [Links](#links)
+
 ## Getting Started
 
 ### Prerequisites
 
-Note: Ensure uv version is >= 0.8.11
-Install [uv](https://docs.astral.sh/uv/getting-started/installation/)
-
-### Installing
+- UV version is >= 0.8.11. Install and upgrade from [docs.astral.sh/uv](https://docs.astral.sh/uv/getting-started/installation/).
+- Python version is >= 3.12.  Install or upgrade from [python.org/downloads](https://www.python.org/downloads/).
 
-1. `uv sync --all-packages --group dev` - it installs the virtual env and dependencies
-   - If you are using Windows, you may need to manually install [cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html)
-2. Activate virtual env
+### Installation
 
-- Mac: `source .venv/bin/activate`
-- Windows: `.venv\Scripts\Activate`
+#### 1. Install the dependencies.
+```bash
+ uv sync --all-packages --group dev
+```
 
+#### 2. Activate the virtual env
 > **Note:** After the initial setup, you need to activate the virtual environment each time you start a new terminal session
+```bash
+# On Mac
+ `source .venv/bin/activate`
+
+# On Windows
+ `.venv\Scripts\Activate`
+```
 
-3. Install pre-commit hooks: `pre-commit install`
+#### 3. Install the pre-commit hooks
+```bash
+ pre-commit install
+```
 
-## Creating a new package
+## Packages
+
+> ℹ️ core packages used to build client/server apps for Teams.
+
+- [`microsoft-teams-apps`](./packages/apps/README.md)
+- [`microsoft-teams-ai`](./packages/ai/README.md)
+- [`microsoft-teams-api`](./packages/api/README.md)
+- [`microsoft-teams-cards`](./packages/cards/README.md)
+- [`microsoft-teams-common`](./packages/common/README.md)
+- [`microsoft-teams-devtools`](./packages/devtools/README.md)
+- [`microsoft-teams-graph`](./packages/graph/README.md)
+- [`microsoft-teams-openai`](./packages/openai/README.md)
```

Comment:
not a huge fan of keeping lists like this, because they inevitably go out of sync. We can just point to the packages folder?